ggshield/__main__.py

```diff
@@ -22,15 +22,15 @@
 from ggshield.cmd.status import status_cmd
 from ggshield.cmd.utils.common_options import add_common_options
 from ggshield.cmd.utils.context_obj import ContextObj
-from ggshield.cmd.utils.debug_logs import disable_logs, setup_debug_logs
-from ggshield.core import check_updates
+from ggshield.cmd.utils.debug import setup_debug_mode
+from ggshield.core import check_updates, ui
```

Comment:
Maybe it's just me but I'm not a fan of the short naming here. When reading, my first impression is that it's just a local variable and not a global object that can be handled and modified in plenty of places, could be confusing in the future...
Maybe another casing or something like `global_ui` could be clearer ?